vizro-core/docs/pages/user-guides/card-button.md

```diff
@@ -396,25 +396,14 @@ and give an attribute selector to select images with that matching URL hash.
 
     [CardImageFloating]: ../../assets/user_guides/components/card_image_floating.png
 
-### Create a navigation card
-
-!!! note
-
-    This section describes how to use the [`Card`][vizro.models.Card] component to create a navigation
-    card. To configure the navigation panel on the left hand side of the screen, refer to the
-    [guide on navigation](navigation.md).
-
-
-A navigation card enables you to navigate to a different page via a click on the card area.
-To create a navigation card, do the following:
+### Add an icon
+If you want to add an icon to your card, add your image as described in the [earlier section](#placing-images)
```

Comment:
I didn't see a "Placing images" subsection in this doc?

Reply:
Ah, interesting. I've just moved this section up, but didn't change the content. I think the markdown references are outdated then 🫠